src/content/article/kodi-21-0-omega-beta-3/index.md

```diff
@@ -0,0 +1,99 @@
+  alt: Kodi 21 "Omega" Splash Screen - the Kodi logo sits at the centre of a black
+    screen, a stylised Greek letter "Omega" around it.
+---
+Time marches on for Kodi 21 "Omega", we are here with the third Beta release!
+
+This is a major release, so, as you'd expect, it comes with many new features. However, with a core change to FFmpeg 6.0, there is a very good chance that some regressions may occur.  As such, please be aware that installing this will most likely break things, and we'd really like your help at that point to identify the problems and get them fixed. Please raise [Github issues](https://github.com/xbmc/xbmc/issues) with full debug logs to help us resolve any issues you wonderful testers come across.
+
+First, we'd like to make a few targeted announcements.
+
+**Android users**
+
+Many users may still be on the Kodi testing track for their Android devices. It's far easier to leave the testing track from a 'phone-based device, but be aware that if your device is on the testing track, we will be continuing to deploy Beta 3 releases over the next week or so. If you wish to opt out of the testing group, one way you can is by visiting the following link [Google Play - Kodi testing track](https://play.google.com/apps/testing/org.xbmc.kodi). From here, you can opt out for your Google Play account. This is the best (and possibly only) way to opt out on many Android TV devices (e.g. Nvidia Shields).
+
+With that said, for those adventurous types who would like to help us with testing, you can use the same link to opt in for a Google Play account.
+
+**Skin Developers**
+
+An API bump has been made for Kodi 21's testing cycle. See the following PRs for details:
+
+https://github.com/xbmc/xbmc/pull/23926
+
+https://github.com/xbmc/xbmc/pull/23862
+
+This will require changes and submissions to the Omega branch for skins for those who submit to the Kodi Repository. Reach out on the forums if you have any questions/concerns.
+
+Please note, https://github.com/xbmc/xbmc/pull/23927 has been merged shortly after Beta 2, and all skins will need to  update the GUI version number to be available in Omega repos for any builds (including nightlies) after Beta 2.
+
+The following forum thread has the skinning change info for Omega: https://forum.kodi.tv/showthread.php?tid=372280&pid=3174566#pid3174566
+
+And now, for the list of changes for Beta 3.
+
+**Build/Dependencies**
+
+
+**Estuary, Estouchy**
+
+**General**
+- The way in which buffer fill was handled was changed to care more for low-power CPU devices. Under certain circumstances, and with especially high manually-configured `readbuffer` factors, the buffer filling thread caused high CPU usage and introduced stutter by itself.
+- Implemented new "Adaptative" setting in Services / Caching / Read Factor that allows use variable read factor based on cache level. Cache fills faster and at same time it uses less aggressive read factor when is already filled.
```

Comment:
```suggestion
- Implemented new "Adaptative" setting in Services / Caching / Read Factor that allows the use of a variable read factor based on cache level. Cache fills faster and at the same time it uses a less aggressive read factor when it is already filled.
```
I've tried to improve the English and made a guess at what was meant, please check.